src/include/migraphx/matcher.hpp

```diff
@@ -1041,6 +1041,14 @@ inline auto has_attribute(const std::string& name)
         [=](instruction_ref ins) { return ins->get_operator().attributes().contains(name); });
 }
 
+inline auto has_op_value(const std::string& name, const std::string& value)
+{
+    return make_basic_pred_matcher([=](instruction_ref ins) {
+        return ins->get_operator().to_value().contains(name) and
```

Comment:
I would assign `ins->get_operator().to_value()` to a variable first then check it, so we dont serialize twice.